x-pack/plugins/reporting/public/lib/reporting_api_client/reporting_api_client.ts

```diff
@@ -128,13 +132,15 @@ export class ReportingAPIClient implements IReportingAPI {
     return reports.map((report) => new Job(report));
   }
 
-  public getReportingJobPath(exportType: string, jobParams: JobParams) {
-    const params = stringify({ jobParams: rison.encode(jobParams) });
+  public getReportingJobPath(exportType: string, jobParams: BaseParams) {
+    const risonObject: RisonObject = jobParams as Record<string, any>;
```

Comment:
why do we need this additional casting now?

Reply:
This is pretty much moving the casting down from the local type which was removed: `interface JobParams`